hack/setup_env.sh

```diff
@@ -5,24 +5,53 @@ set -o pipefail
 set -o errexit
 set -o xtrace
 
-curl --retry 5 -L "https://dl.k8s.io/release/$(curl --retry 5 -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl" -o /tmp/kubectl && \
+function print_help() {
+  ALL_FUNCS="kustomize|golang|assisted_service|hive_from_upstream|print_help"
+  echo "Usage: bash ${0} (${ALL_FUNCS})"
+}
+
+function kustomize() {
+  (cd /usr/bin &&
+    curl --retry 5 -s "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh" \
+      | bash -s -- 3.8.8)
+}
+
+function golang() {
+  echo "Installing golang..."
+  curl -L https://storage.googleapis.com/golang/getgo/installer_linux -o /tmp/golang_installer
+  chmod u+x /tmp/golang_installer
+  /tmp/golang_installer
+  rm /tmp/golang_installer
+
+  echo "Activating go command on current shell..."
+  set +u
+  source /root/.bash_profile
+  set -u
+}
+
+function assisted_service() {
```

Comment:
maybe `all`?

Reply:
it's not including ``golang`` :)

Reply:
why not?

Reply:
it assumes golang already installed, and it's indeed the case in the image ``registry.ci.openshift.org/openshift/release:golang-1.15`` we're consisting on

Reply:
also, it's the better name
we don't want to change it if we'll add more installations here that ``assisted-service`` doesn't need